user-journey-tests/helpers/page.js

```diff
@@ -91,3 +91,15 @@ export const clearElement = async (element) => {
     )
   }
 }
+
+export const checkForSecurityPopUpAndResolve = async () => {
+  const contexts = await browser.getContexts()
+  if (contexts.length > 1) {
```

Comment:
As this is checking whether we have more than one context, I wonder if you can remove the check for device name and run them all past this function?

Reply:
It only recognises this contexts function in browserstack so otherwise it errors the test because it doesn't recommend the command

Reply:
browser.getContexts is not a function